extensions/amp-autocomplete/0.1/amp-autocomplete.js

```diff
@@ -512,13 +561,21 @@ export class AmpAutocomplete extends AMP.BaseElement {
   renderResults_(filteredData, container, input) {
     let renderPromise = Promise.resolve();
     this.resetActiveElement_();
-    if (this.templateElement_) {
-      renderPromise = this.templates_
-        .renderTemplateArray(this.templateElement_, filteredData)
-        .then(renderedChildren => {
-          renderedChildren.map(child => {
+    if (this.hasTemplate_) {
+      renderPromise = this.ssrTemplateHelper_
+        .applySsrOrCsrTemplate(this.element, filteredData)
+        .then(rendered => {
+          const elements = isArray(rendered)
+            ? rendered
+            : [].slice.call(rendered.children);
```

Comment:
While direct usage of `slice` is idiomatic, should use [`toArray()`](https://github.com/ampproject/amphtml/blob/6b2c20d0320d2565424c19b63abee60a81757add/src/types.js#L44-L46) instead to match the rest of the code base.